src/OrbitDB.js

```diff
@@ -41,11 +56,12 @@ let databaseTypes = {
     this.identity = identity
     this.id = options.peerId
     this._pubsub = options && options.broker
-      ? new options.broker(this._ipfs)
+      ? new options.broker(this._ipfs) // eslint-disable-line
       : new Pubsub(this._ipfs, this.id)
     this.directory = options.directory || './orbitdb'
+    this.storage = options.storage
```

Comment:
Do we need to pass in `storage` in addition to `keystore` and `cache`? Can this be handled in`createInstance` and only pass the latter?